src/JoomlaBrowser.php

```diff
@@ -157,6 +156,7 @@ public function installJoomla()
         $I->click(['xpath' => "//fieldset[@id='jform_site_offline']/label[@for='jform_site_offline1']"]); // ['No Site Offline']
         $this->debug('I click Next');
         $I->click(['link' => 'Next']);
+        $I->wait(1);
```

Comment:
we are waiting in line 162 for the Text to appear, I think this one is not needed.